components/motor/gpio/motor_encoder.go

```diff
@@ -642,5 +642,5 @@ func (m *EncodedMotor) Close(ctx context.Context) error {
 	}
 	m.cancel()
 	m.activeBackgroundWorkers.Wait()
-	return m.real.Stop(ctx, nil)
+	return nil
```

Comment:
this also caused an issue not sure why, changed it back to nil as this also doesn't seem to be the true fix to the original close of closed channel bug